app/controllers/idv/by_mail/request_letter_controller.rb

```diff
@@ -5,7 +5,6 @@ module ByMail
     class RequestLetterController < ApplicationController
       include Idv::AvailabilityConcern
       include IdvStepConcern
-      skip_before_action :confirm_no_pending_gpo_profile
```

Comment:
This changes breaks a lot of tests related to sending a letter. It won't work until I get all of the changes up to #10866 merged and rebase

Reply:
I changed the base for this to the branch for #10866. I'll change it to main when this is merged.